db/migrate/20220117192314_create_curiosity_cards.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCuriosityCards < ActiveRecord::Migration[7.0]
+  def change
+    create_table :curiosity_cards do |t|
+      t.string :title
```

Comment:
At model level you're validating the presences of both attributes `title` and `content`. If a record shouldn't be valid without any of those, I'd add the validation at db level as well

Reply:
:o interesting, how can I do that?